app/scripts/controllers/permissions/index.js

```diff
@@ -73,6 +86,7 @@ class PermissionsController {
       getAccounts: this.getAccounts.bind(this, origin),
       requestPermissions: this._requestPermissions.bind(this, origin),
       installPlugins: this.installPlugins.bind(this),
+      getPlugins: this.getPermittedPlugins.bind(this),
```

Comment:
You probably meant to also bind the `origin` here, as to not trust the method caller to provide its origin?

Reply:
See: https://github.com/MetaMask/metamask-snaps-beta/pull/137#discussion_r359989304